### ISSUE TITLE
No way to remove `TempSensorBreadcrumbsIntegration` (in new versions), causing app *rejected* by app stores

### ISSUE THREAD

**Issue by fzyzcjy** [getsentry/sentry-java#2821]
### Integration

sentry-android

### Build System

Gradle

### AGP Version

latest

### Proguard

Enabled

### Version

latest

### Steps to Reproduce

Last time I submitted https://github.com/getsentry/sentry-java/issues/2355 which caused apps rejected by stores, the solution was to do something like:

```
SentryAndroid.init(context) { options ->
  options.integrations.removeAll { it is TempSensorBreadcrumbsIntegration }
}
```

However, that does not work after I upgrade Sentry. Indeed, that issue was solved at 2022.11.14, while the code that breaks the solution was created at 2022.11.18 (what a short time!) at https://github.com/getsentry/sentry-java/pull/2377.

As you can see, with that PR, the configure callback is executed *before* the TempSensorBreadcrumbsIntegration has a chance to be added to the integrations.

https://github.com/getsentry/sentry-java/blob/f274c795d1c562858cf44f547b8c1e69bc2e0bee/sentry-android-core/src/main/java/io/sentry/android/core/SentryAndroid.java#L111-L113

Therefore, I would appreciate it if this could be fixed. For example, we can add something similar to `isEnableSystemEventBreadcrumbs`, but specific to temperature sensor.

### Expected Result

-

### Actual Result

-

**Comment by stefanosiano** [comment]
Hey @fzyzcjy this issue was fixed in the release `6.25.1`.
Feel free to reopen this issue in case something goes wrong

**Comment by fzyzcjy** [comment]
Thank you, it looks great!
